docs/source/markdown/podman-secret-rm.1.md

```diff
@@ -22,6 +22,8 @@ the old secret value still remains.
 
 Remove all existing secrets.
 
+@@option ignore
```

Comment:
Weird, @rhatdan is this the wrong usage for referencing an option? Or does the header needs updating.

Reply:
Yes you need to 
git mv docs/source/markdown/podman-secret-rm.1.md docs/source/markdown/podman-secret-rm.1.md.in

And then it should work correctly

Reply:
I am not sure if there are other steps.